### DISCUSSION TITLE
Try to use clean zone with Lydsto r1 vacuum cleaner

### DISCUSSION THREAD

**Discussion by sagidam** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#399] [Q&A]
I am using Xiaomi Miot For HomeAssistant to integrate the vacuum cleaner
evrything work very good excpet the clean zone 
i am try to use [demo config](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/demo_config.yaml#L124), but getting the error "service not found" for clean zone, 
How i can fix it?

**Comment by PiotrMachowski** [comment]
You have already asked this question [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/393)...
